### PR TITLE
Add option for accessing local media uploads from KoboCat within KPI

### PR DESCRIPTION
**Pull request by yahyazini**

### CODE REVIEW

File: docker-compose.local.yml [hunk 1]
```diff
@@ -106,6 +106,8 @@ kpi:
     # Dev: Use live source directories from the host machine.
     # - ../kpi:/srv/src/kpi
     #- ../formpack/src/formpack:/usr/local/lib/python2.7/dist-packages/formpack
+    # In case you're developping locally and solely using local storage uncomment the following line so you can access media uploads locally within KPI
+    # - ./.vols/kobocat_media_uploads:/srv/src/kobocat/media
```
Comment (jnm) [inline docker-compose.local.yml]: @yahyazini, this isn't just for local development: we have some (security conscious) organizations who refuse to use services like S3 and store all their media locally. Could you enable the volume mapping by default and also add it to `docker-compose.server.yml`?

File: docker-compose.local.yml [hunk 2]
```diff
@@ -125,7 +127,7 @@ nginx:
   environment:
     - NGINX_CONFIG_FILE_NAME=nginx_site_http.conf
   ports:
-    # NOTE: The host ports (left of the ":") must match the values in 
+    # NOTE: The host ports (left of the ":") must match the values in
```
Comment (jnm) [inline docker-compose.local.yml]: @yahyazini, you're not wrong to want to remove these trailing spaces, but I'd request you do this kind of cleaning only on lines that you're already editing. For example, if you thought the comment here was inaccurate or needed to be reworded, then by all means remove the sloppy whitespace while you're at it.

When editors automatically make style corrections en masse (e.g. stripping whitespace or reordering all the `import` statements) and these changes are committed, it muddies the `git blame` and clutters up PRs, making them more difficult to review.

Comment (yahyazini) [inline docker-compose.local.yml]: Sorry about that, my editor removed trailing spaces for Python by default, I just disabled it.

**Comment by yahyazini** [comment]
@jnm I just made those edits and going forward those trailing spaces should not be removed.
Thanks for the help!

**Pull request merged**